PhysicsTools/Heppy/python/analyzers/objects/autophobj.py

```diff
@@ -53,6 +53,7 @@
     # Isolations with the two radia
     NTupleVariable("relIso03",  lambda x : x.relIso03, help="PF Rel Iso, R=0.3, pile-up corrected"),
     NTupleVariable("relIso04",  lambda x : x.relIso04, help="PF Rel Iso, R=0.4, pile-up corrected"),
+    NTupleVariable("miniRelIso",  lambda x : x.miniRelIso if hasattr(x,'miniRelIso') else  -999, help="PF Rel miniRel, pile-up corrected"),
```

Comment:
You added it here, but it was already in `leptonTypeSusy`. If you think it should be here, then remove it from there to avoid duplicates.